Project.toml

```diff
@@ -14,6 +14,10 @@ OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 
 [compat]
-julia = "^1.2"
+julia = "^1.7"
```

Comment:
Does GuidingCenterOrbits.jl require Julia 1.7 an upwards now? Is there anything in the package that requires 1.7?

Reply:
New StaticArrays 1.5.0 needs julia 1.6, as does JLD2 vers 0.4.22. I could put compatibility to 1.6, I've been running it on 1.7 so that's the only version I was sure of.

Reply:
Ok easy ill put 1.6